sample-cf-install-values.yml

```diff
@@ -30,18 +30,18 @@ system_certificate:
 uaa:
   admin_client_secret: uaaadminclientsecret
   certificate:
-    #! This certificate should be valid for uaa.cf-system.svc.cluster.local
-    crt: *crt
-    key: *key
+    #! This certificate should be valid for uaa.cf-system.svc.cluster.local and signed with the same ca as in system_certificate.ca
+    crt: &cluster_crt LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUUrRENDQTJDZ0F3SUJBZ0lRUFBrOHR1YVlWRGdEVTh1amVCU0tIVEFOQmdrcWhraUc5dzBCQVFzRkFEQXoKTVF3d0NnWURWUVFHRXdOVlUwRXhGakFVQmdOVkJBb1REVU5zYjNWa0lFWnZkVzVrY25reEN6QUpCZ05WQkFNVApBbU5oTUI0WERUSXdNREV5TnpJd01qWXlNbG9YRFRJeE1ERXlOakl3TWpZeU1sb3dVREVNTUFvR0ExVUVCaE1EClZWTkJNUll3RkFZRFZRUUtFdzFEYkc5MVpDQkdiM1Z1WkhKNU1TZ3dKZ1lEVlFRRERCOHFMbk41YzNSbGJTNWoKYkc5dVpTNWtaWFl1Y21Wc2FXNTBMbkp2WTJ0ek1JSUJvakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBWThBTUlJQgppZ0tDQVlFQXU1M0E4eTNBQUMrMHRCNXJ3UWY0ckd3WnVOUkV1OW1OYk1oSFFpRnJnNk92Y1VGSzlBRFAyWXVuClJjNkJjeXg1S3ZFMHVlK2VOcWxobUVVckdzS0FlVVRZNGFFRkdpNmF2TzFxTmY5VHdDZW1JTWxGZklGaDhkdDkKY21HOXpIS0tWN1hRRXVWd2kxYlU3b0cwQjd3QWJXeTJQWHpVdFo5YWhTT1l0aTUxdUVzdmRYV29HTmIyRzVIUQptanQyMEdQcGwrSWdsb1ZiUEN0bUxtei9yTVF4cDZPdGVrZ1F4Vy85Y01HY1ZzZlRjc0dhdExQbUVzaUMzdk1QCmRqU0Qwck1TNkppUGtJVmlhZkQ2WlJkeFBDNjFYS04rUGczQ3V3ajUzMnRoVjBFMjllamF2WUJDYi9nN1h0S2YKdnhmSkgyZE4zYTUvVjRIVXRZYkZpUS9nblJqMzFWL05VdnZFcVJkWFNJckNpM2tTcldnb2l6VExJSVVsTmFEdApiVzJ5NG05RnpuZjl2UzZ2OUxIbktSVC9kV0ZqdFRIV0JFdTV5NHdFdHU1MW9uYWJNZFA0RnJFR1dEaGJoOXV4CmF1eFFIZDN0V3A1bVp3QXUxRVMySG1RK05lYzhPZjd0UkZRL0ZiSjVqM21SVHlJSFlJUTNSMXFiNjd4aFRSWlIKNnZSMGZZeUZBZ01CQUFHamdlb3dnZWN3RGdZRFZSMFBBUUgvQkFRREFnV2dNQjBHQTFVZEpRUVdNQlFHQ0NzRwpBUVVGQndNQ0JnZ3JCZ0VGQlFjREFUQU1CZ05WSFJNQkFmOEVBakFBTUIwR0ExVWREZ1FXQkJUQVFhQ2J2NUMzCkVRb2x5VldKZ1dUMklLT24wekFmQmdOVkhTTUVHREFXZ0JTNTMrVmhGMkZFR2NGS3FkNDJlZkRiUEYrNm96Qm8KQmdOVkhSRUVZVEJmZ2g4cUxuTjVjM1JsYlM1amJHOXVaUzVrWlhZdWNtVnNhVzUwTG5KdlkydHpnaDBxTG1GdwpjSE11WTJ4dmJtVXVaR1YyTG5KbGJHbHVkQzV5YjJOcmM0SWRLaTVqWmkxemVYTjBaVzB1YzNaakxtTnNkWE4wClpYSXViRzlqWVd3d0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dHQkFHM1FKc0tWMmtOT3N6NVNNcUV2K2ZDRjEyR0QKVmNhN1pBeVhKYXdobnNCVFphemRnYzRVWHM1WllqZ3RlQjZjQ3VUY3VPMmFzV3ZrYVFUa1FobThHZC9adHVWQgpHTS9Fb3l2dUJZRktiNGx6d0V3U2NxcXVJSVo0RlNnWVhIcHdpNVh4cDcra01ueU9QM3FmTlBoR0FnMHNXWmlzCndHU3M2SGZwRTJMd0lUc2ZpbFgrNmZYMXBPdklNaUw1TWN2bHVUdkNOcG95TCs3UUNFY1N0eWFoQm9zUGhPb1MKVGNRZ1R0cStrSlNCa2VsRUYySVpJaWp6T1QzUFRjejJrdW1TZTFrMXVTQmFFcldLRkt0OG1vdFkyMUUya1RsbAp2bVpmYVh5UWlWWVE2cktUKzJpUXpoSmJScHpFT0pWNjkwVFBQWFl2ZnRjc1R2VkZ5RDkrY3gzS05PL1VxeVF4CjAvNElHc1BtMVRKRStZNzU1aXd6RTY3ZlVCSFgzZTZjK3NWdFZGYWlOY2ZraTZWZ3VFSG8vaWhGeWs3WlNtdnkKRFBHR3VHMVlXSW1nMURBWmJBZGdsSTBSQ05Ta1ZoNFNZbmdDREMxVDVkOTNobUF3N3BJLzZnTjJ2V0xVWTddaeUxnbE9zRzhqaUNRRTBBWHpjbEZSSEh5VUNZVE9FNXMxQT09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0KCg
```

Comment:
@mike1808 How do you feel about renaming the anchors to be more explicit along the lines of: `internal_system_crt` and `internal_system_key`?